featuretools/feature_base/feature_base.py

```diff
@@ -520,21 +512,26 @@ def __init__(self, base_features, parent_entity, primitive,
 
     def _handle_relationship_path(self, parent_entity, relationship_path):
```

Comment:
The goal here would be to make it easier for the end user to directly construct features? We had also discussed allowing a string to be passed instead of a path (like "customers.transactions"), which would probably be the easiest API for the end user. If we are planning on that is it worth also supporting passing a list of relationships?

Reply:
i see. let's hold off on this for now